ports/libfaketime/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(libfaketime VERSION 0.9.10)
+
+add_library(
+        ${PROJECT_NAME}
+        src/libfaketime.c
+)
+
+set_target_properties(${PROJECT_NAME} PROPERTIES PREFIX "")
+
```

Comment:
```suggestion
```

I don't think this is necessary.

Reply:
If this is not present the produced library is named `liblibfaketime.so`

Reply:
I could rename the library target to `faketime
`

Reply:
Yeah that sounds better.